src/js/load.js

```diff
@@ -65,6 +67,7 @@ export default function configureApplication(profileId, config) {
     const boundaryTypeBox = new BoundaryTypeBox(config.config.preferred_children);
     const mapDownload = new MapDownload(mapchip);
     const tutorial = new Tutorial();
+    const tabNotice = new TabNotice();
```

Comment:
the feedback is on the profile configuration. 
so it can and should be added here.

`const tabNotice = new TabNotice(config.config.feedback);`

than a whole lot of other code can be deleted.

Reply:
updated my code